src/chapters/loops/while-loops.rst

```diff
@@ -143,20 +143,16 @@ Do the following in the editor below:
 Input Validation
 ----------------
 
-``while`` loops require a little more effort to code than ``for`` loops, but
-``while`` loops tend to be more flexible.
-
-We can replace ANY ``for`` loop with a ``while``, but the reverse is NOT true.
-Many ``while`` loops can be converted into ``for`` loops, but there are some
-instances where only a ``while`` loop will work. Let's look at one example of
-this.
```

Comment:
I don't think the replacement paragraph negates the concept check questions. I removed this paragraph because it undersold ``while`` loops and why we use them.

Reply:
OK, I'm confused now.  I think I missed the replacement paragraph (and I'm not sure I've found it yet).

Still, when I re-read the page, the for-replaced-by-while idea slaps me in the face, but I'm not sure if the while-not-into-for idea is explicitly stated.  That's why I was unsure about the first two CC questions.

Also, when we merge the "Which Loop" page into this one, we will need to reconsider the CC questions anyway.